spec/fixtures/requests/mutations/photo_create.graphql

```diff
@@ -0,0 +1,7 @@
+mutation {
+    photoCreate(imageableId: 1, imageableType:"Freak", uri: "https://www.gravatar.com/avatar/00000000000000000000000000000000") {
```

Comment:
So we don't care if the imageableType+imageableId actually exist in the database.